bin/rector

```diff
@@ -42,6 +45,24 @@ try {
     $rectorConfigsResolver = new RectorConfigsResolver();
     $configFileInfos = $rectorConfigsResolver->provide();
 
+    // report YAML deprecated configs
+    foreach ($configFileInfos as $configFileInfo) {
+        if (! in_array($configFileInfo->getSuffix(), ['yml', 'yaml'], true)) {
+            continue;
+        }
+
+        $warning = sprintf(
+            'You are using YAML format in "%s" config.%sIt will be removed in Rector 0.8. Switch to PHP format as soon as possible with "%s"',
+            $configFileInfo->getRelativeFilePathFromCwd(),
+            PHP_EOL,
+            'https://github.com/migrify/config-transformer'
+        );
+
+        $symfonyStyle->warning($warning);
+
+        sleep(3);
```

Comment:
Why this?

Reply:
This is an old trick, how to deliver the message the feature is removed.

The message would be overriden by new content of the tool. This makes sure it's visible and noticed.

Otherwise might happen people overlook it, and in next version they'll be surprised by new bugs of missing YAML support.